File: ACAD_menu/app/src/main/java/com/afiq/acad_menu/MainActivity.java
package com.afiq.acad_menu;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.widget.Toast;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
    }

    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.main_menu, menu);
        return true;
    }

    public boolean onOptionsItemSelected(MenuItem item) {
        int itemid =  item.getItemId();

            if(itemid == R.id.message){
                toast("message");
            }else if (itemid == R.id.about){
                toast("about");
            }else if(itemid == R.id.exit){
                finish();
            }else if(itemid == R.id.picture){
                toast("picture");
            }else{
                toast("add anything else");
            }
       return true;
    }
    void toast(String message){
        Toast.makeText(getApplicationContext(), message, Toast.LENGTH_SHORT).show();
    }
}

File: MaskEdit/app/src/main/java/com/example/maskedit/MainActivity.kt
package com.example.maskedit

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import com.vicmikhailau.maskededittext.MaskedEditText

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val edt1 = findViewById<MaskedEditText>(R.id.edtCreditCard)
        val edt2 = findViewById<MaskedEditText>(R.id.edtPhoneNum)

        val myBtn = findViewById<Button>(R.id.btn)

        myBtn.setOnClickListener {
        val text1 = edt1.text.toString()+" "+edt2.text.toString()
            Toast.makeText(applicationContext,text1,Toast.LENGTH_LONG).show()
        }

    }
}